terraform-module/modules/frontend-spa-s3/main.tf

```diff
@@ -17,9 +17,12 @@ resource "aws_s3_bucket" "origin" {
   }
 }
 
-resource "aws_s3_bucket_acl" "origin" {
+resource "aws_s3_bucket_ownership_controls" "origin" {
   bucket = aws_s3_bucket.origin.id
-  acl    = null
+
+  rule {
+    object_ownership = "ObjectWriter"
+  }
 }
```

Comment:
What is the terraform plan for this change?

Reply:
```
# module.app.module.s3.aws_s3_bucket_acl.origin will be destroyed
# (because aws_s3_bucket_acl.origin is not in configuration)
- resource "aws_s3_bucket_acl" "origin" {
    - acl    = "private" -> null
    - bucket = "pleo-product-web-origin-staging" -> null
    - id     = "pleo-product-web-origin-staging,private" -> null

    - access_control_policy {
        - grant {
            - permission = "FULL_CONTROL" -> null

            - grantee {
                - display_name = "admin-aws+product-staging" -> null
                - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
                - type         = "CanonicalUser" -> null
              }
          }
        - owner {
            - display_name = "admin-aws+product-staging" -> null
            - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
          }
      }
  }

# module.app.module.s3.aws_s3_bucket_ownership_controls.origin will be created
+ resource "aws_s3_bucket_ownership_controls" "origin" {
    + bucket = "pleo-product-web-origin-staging"
    + id     = (known after apply)

    + rule {
        + object_ownership = "ObjectWriter"
      }
  }

# module.storybook[0].module.s3.aws_s3_bucket_acl.origin will be destroyed
# (because aws_s3_bucket_acl.origin is not in configuration)
- resource "aws_s3_bucket_acl" "origin" {
    - acl    = "private" -> null
    - bucket = "pleo-product-web-storybook-origin-staging" -> null
    - id     = "pleo-product-web-storybook-origin-staging,private" -> null

    - access_control_policy {
        - grant {
            - permission = "FULL_CONTROL" -> null

            - grantee {
                - display_name = "admin-aws+product-staging" -> null
                - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
                - type         = "CanonicalUser" -> null
              }
          }
        - owner {
            - display_name = "admin-aws+product-staging" -> null
            - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
          }
      }
  }

# module.storybook[0].module.s3.aws_s3_bucket_ownership_controls.origin will be created
+ resource "aws_s3_bucket_ownership_controls" "origin" {
    + bucket = "pleo-product-web-storybook-origin-staging"
    + id     = (known after apply)

    + rule {
        + object_ownership = "ObjectWriter"
      }
  }
```

Reply:
Any thoughts? 😄